### ISSUE TITLE
Unable to use with Rails 7

### ISSUE THREAD

**Issue by cjavdev** [posiczko/devise-tailwindcssed#15]
Wonder if we could bump the dependency and if it'd work with Rails 7, especially given the new `-c tailwind` option for new rails apps.

```
The dependency tzinfo-data (>= 0) will be unused by any of the platforms Bundler is installing for. Bundler is installing for ruby but the dependency is only for x86-mingw32, x86-mswin32, x64-mingw32, java. To add those platforms to the bundle, run `bundle lock --add-platform x86-mingw32 x86-mswin32 x64-mingw32 java`.
Fetching gem metadata from https://rubygems.org/...........
Fetching gem metadata from https://rubygems.org/.
Resolving dependencies...
Bundler could not find compatible versions for gem "rails":
  In snapshot (Gemfile.lock):
    rails (= 7.1.0.alpha)

  In Gemfile:
    rails

    devise-tailwindcssed was resolved to 0.1.0, which depends on
      rails (<= 6.0, ~> 5.2.3)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

**Comment by danieldenis01** [comment]
I solved this issue through PR #16, changing gem's compability resctrictions.

**Comment by posiczko** [comment]
Thanks to @danieldenis01 for the PR. Closing.
